README.md

```diff
@@ -19,15 +24,40 @@ This project does not:
 * Replace [Helm](https://github.com/kubernetes/helm)
 * Turn Kubernetes into a [PaaS](https://en.wikipedia.org/wiki/Platform_as_a_service)
 
+## Prerequisites
+
+- [git][git_tool]
+- [go][go_tool] version v1.12+.
+- [mercurial][mercurial_tool] version 3.9+
```

Comment:
Why is mercurial a requirement? should it be listed as an alternative for git?

Reply:
👍 good spot. Thank you for your time and review. (removed :-))